src/Dapr.AspNetCore/StateEntryApplicationModelProvider.cs

```diff
@@ -33,7 +33,7 @@ public void OnProvidersExecuted(ApplicationModelProviderContext context)
                     }
                     else if (IsStateEntryType(property.ParameterType))
                     {
-                        property.BindingInfo.BindingSource = new FromStateBindingSource(null);
+                        property.BindingInfo.BindingSource = new FromStateBindingSource(null, null);
```

Comment:
This is the piece of code that has to change... We can't use a state store without providing a store name. You should change this if block to throw an exception explaining that `[FromState(...)]` is required.

Reply:
@ryanvolum Does that mean that following cannot be written without a FromState attribute?
```C#
[HttpGet("{account}")]
public ActionResult<Account> Get(StateEntry<Account> account)
```

It must always be written as:
```C#
[HttpGet("{account}")]
public ActionResult<Account> Get([FromState("storeName")]StateEntry<Account> account)
```

Reply:
correct. The store name is required.